packages/xo-web/src/common/intl/messages.js

```diff
@@ -2307,6 +2307,8 @@ const messages = {
   proxyLinkedBackups: 'Click to see linked backups',
   proxyNetworkDnsPlaceHolder: 'Default to: {dns}',
   proxyNetworkNetmaskPlaceHolder: 'Default to: {netmask}',
+  proxySrPredicateInfo:
+    'The selected SR should be at least connected to an HVM-capable host',
```

Comment:
... to a HVM-capable host

Reply:
Why?

Reply:
I think, we say `a HVM` not `an HVM` ^_^

Reply:
IMHO, we say `an HVM` because the H [sounds like a vowel](https://www.englishclub.com/pronunciation/a-an.htm).